fdbserver/IPager.h

```diff
@@ -40,6 +40,45 @@ typedef uint32_t PhysicalPageID;
 typedef uint32_t QueueID;
 #define invalidQueueID std::numeric_limits<QueueID>::max()
 
+// Pager Events
+enum class PagerEvents { pagerCacheLookup = 0, pagerCacheHit, pagerCacheMiss, pagerWrite, MAXEVENTS };
+static const std::string PagerEventsCodes[] = { "Lookup", "Hit", "Miss", "Write" };
+// Reasons for page level events.
+enum class PagerEventReasons { pointRead = 0, rangeRead, rangePrefetch, commit, lazyClear, metaData, MAXEVENTREASONS };
+static const std::string PagerEventReasonsCodes[] = { "Get", "GetR", "GetRPF", "Commit", "LazyClr", "Meta" };
+
+static const int nonBtreeLevel = 0;
+static const int rootLevel = 1;
```

Comment:
Remove this because `rootLevel` is not a constant.  I commented in the places where it is used on how to get the actual root level.